services/QuillLMS/spec/models/subscription_spec.rb

```diff
@@ -202,6 +202,15 @@
     end
   end
 
+  describe '#length_in_months' do
+    it 'returns the length of the subscription in months' do
+      length_in_months = 45
+      subscription = create(:subscription, start_date: Time.zone.today, expiration: Time.zone.today + (length_in_months).months)
```

Comment:
I don't think the parens around `length_in_months` is necessary here, but I actually like it as a way to emphasize where the `months` call will apply.  (I often get confused about how Ruby is going to distribute some of those method calls.)